modules/channel-vonage/src/backend/client.ts

```diff
@@ -109,23 +111,58 @@ export class VonageClient {
 
     const conversation = await this.conversations.recent(userId)
 
+    const messageContent = <ExtendedChannelContent>body.message.content
+
     // https://developer.nexmo.com/api/messages-olympus?theme=dark
     let payload: sdk.IO.IncomingEvent['payload'] = null
-    switch (body.message.content.type) {
+    switch (messageContent.type) {
       case 'text':
-        const text = body.message.content.text
+        const text = messageContent.text
         payload = { type: 'text', text }
 
         // Since single choice and carousel are down rendered, we have to handle option selection differently
         payload = (await this.handleOptionResponse(text, userId, conversation.id)) ?? payload
         break
       case 'audio':
-        // TODO: Link received from Nexmo/Vonage API are only valid for 10min
-        const audio = body.message.content.audio.url
-        payload = { type: 'audio', audio }
+        payload = {
+          type: 'voice',
+          url: messageContent.audio.url
+        }
+        break
+      case 'image':
+        payload = {
+          type: 'file', // TODO: Change image content-type type to 'image' instead of 'file'
+          title: messageContent.image.caption,
+          url: messageContent.image.url,
+          collectFeedback: false
+        }
+        break
```

Comment:
Since we're still on content renderers we have no typings here. Eventually we'll be able to replace all this code here with just `bp.render.image(...)`, `bp.render.video()`, `bp.render.document(...)`, `bp.render.audio(...)`, `bp.render.location(...)`

Right now we don't have content types for locations and documents so we should put a todo to add them eventually (so we can receive them in the same format that we send them).